src/static/js/modules/youtube.js

```diff
@@ -0,0 +1,177 @@
+/* ==========================================================================
+   Youtube Initialization
+   Used on at least `/the-bureau/`.
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+  return $( '.youtube-player_container' ).createYoutube();
+}
+
+// Youtube constructor.
+// @param element {Element} DOM Element.
+// @param options {Object} Object used to customize Youtube.
+// Returns {Object} An Youtube instance.
+function Youtube( element, options ) {
+  this.$element = $( element );
+  this.options = $.extend( true, {}, this.defaults, options );
+  this.init();
+}
+
+// Youtube methods and shared properties
+Youtube.prototype = {
+
+  constructor: Youtube,
+
+  defaults: {
+     playerOptions : {
+          height: '410',
+          width: '640',
+          videoId: 'en0Iq8II4fA',
```

Comment:
Should we default the ID? What if we had a check for the ID existence early on and return an error if one doesn't exist?